src/Native/System.Private.CoreLib.Native/pal_environment.cpp

```diff
@@ -30,3 +30,17 @@ extern "C" void CoreLibNative_Exit(int32_t exitCode)
 {
     exit(exitCode);
 }
+
+extern "C" char** CoreLibNative_EnvironGetSystemEnvironment()
+{
+    char** sysEnviron;
+
+#if HAVE__NSGETENVIRON
+    sysEnviron = *(_NSGetEnviron());
+#else   // HAVE__NSGETENVIRON
+    extern char **environ;
```

Comment:
I know you have copied it from CoreCLR. But it may better to have include `#include <unistd.h>` at the top of the file instead of `extern char **environ;`.  (https://en.cppreference.com/w/cpp/utility/program/getenv).

Reply:
I added the include, but as far as I can tell I'm still required to reference `extern char **environ;`. If I don't, compile is not happy:
```
/Users/robertmiles3/Desktop/corert/src/Native/System.Private.CoreLib.Native/pal_environment.cpp:43:18: error: use of
      undeclared identifier 'environ'
    sysEnviron = environ;
                 ^
```

Reply:
Ok, I see that this is complicate. I think you delete the include and just use `extern char **environ;` like CoreCLR.